services/sky/compositor/layer_host.cc

```diff
@@ -83,6 +83,15 @@ void LayerHost::BeginFrame() {
     root_layer_->Display();
   }
 
+  // We may have culled the root layer down to nothing which is equivalent to
+  // the empty size case above.
+  if (!root_layer_->HaveTexture()) {
+    base::MessageLoop::current()->PostTask(
```

Comment:
To avoid spinning in the case when dart code continually signals that it wants a new frame and then doesn't draw anything we would need a signal to key off of to indicate when we should submit another frame even if we haven't submitted one.  We don't currently have such a signal in the system (although we definitely need to add one).